online_meetings/canvas_zoom_meetings.py

```diff
@@ -99,22 +100,19 @@ def get_zoom_details(self, url: str, data: Dict[str, str], course_id: int):
                 if zoom_json:
                     for meeting in zoom_json["list"]:
                         self.zoom_courses_meetings.append({
-                            'course_id': course_id,
+                            'lti_placement_id': lti_placement_id,
                             'meeting_id': meeting['meetingId'],
-                            'meeting_number': meeting['meetingNumber'],
                             'host_id': meeting['hostId'],
-                            'topic': meeting['topic'],
-                            'join_url': meeting['joinUrl'],
                             'start_time': meeting['startTime'],
                             'status': meeting['status'],
-                            'timezone': meeting['timezone']
```

Comment:
I know #100 says many of these columns shouldn't be included in the report.  Couldn't the report just not use this data?  If there's some other good use for this data, it could be kept.

Reply:
We're removing this data completely is removed because of a discussion with IA on a recent call where they said (and I'm paraphasing): If you aren't actively using PII stored data you shouldn't be storing it, otherwise it's a security concern.

Join URL, Topic and Meeting number were all considered to be PII for Zoom. timezone was just all the same for everyone. I was just retaining the minimal data useful for generating the report.